solutions/platinum/cses-1740.mdx

```diff
@@ -0,0 +1,85 @@
+---
+id: cses-1740
+source: CSES
+title: Intersection Points
+author: Rameez Parwez
+---
+
+[Offical Editorial](https://usaco.guide/CPH.pdf#page=286)
```

Comment:
ok i think not writing ur own explanation is perfectly fine here, but maybe consider adding some comments